src/localization/locales/en-US.json

```
@@ -435,7 +435,7 @@
         "boundary_success": "Boundary was updated",
         "form_development_problem_situtation_helper_text": "Examples of challenges include deforestation, poverty, lack of economic opportunities and funding. <1>Learn more about assessment of landscape challenges</1>.",
         "form_development_problem_situtation_helper_text_url": "https://terraso.org/guide/shared-understanding/assessment-of-landscape-challenges-and-opportunities/",
-        "form_development_intervention_strategy_helper_text": "Examples include convening landscape partners to identify  solutions, conducting policy campaigns and building capacity of stakeholders. Learn more about <1>landscape action plan</1> and <1>finance strategy</2>.",
+        "form_development_intervention_strategy_helper_text": "<0>Examples include convening landscape partners to identify  solutions, conducting policy campaigns and building capacity of stakeholders.</0> <1>Learn more about <1>landscape action plan</1> and <1>finance strategy</2>.</1>",
```

suggestion: I think there is a mismatch here. Should it be this? How do the number tags work?

```suggestion
        "form_development_intervention_strategy_helper_text": "<0>Examples include convening landscape partners to identify  solutions, conducting policy campaigns and building capacity of stakeholders.</0> <2>Learn more about <1>landscape action plan</1> and <1>finance strategy</1>.</2>",
```

---

What I had was incorrect. Yours wasn't quite right, but I have it fixed now.